CHANGELOG.md

```diff
@@ -5,7 +5,25 @@ All notable changes to OpenGHG will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/openghg/openghg/compare/0.7.1...HEAD)
+## [Unreleased](https://github.com/openghg/openghg/compare/0.8.0...HEAD)
+
+## [0.8.0] - 2024-03-19
+
```

Comment:
We should add a quick summary here that is a breaking change and we no longer support object stores generated with earlier versions of openghg.